docs/tempo/website/grafana-agent/tail-based-sampling.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Tail-based sampling"
+---
+
+# Tail-based sampling
+
+Tempo aims to provide an inexpensive solution that makes 100% sampling possible.
+However, sometimes constraints will make a lower sampling percentage necessary or desirable,
+such as runtime or egress traffic related costs.
+Probabilistic sampling strategies are easy to implement,
+but also run the risk of discarding relevant data that you'll later want.
+
+In tail-based sampling, sampling decisions are made at the end of the workflow.
+The Grafana Agent groups spans by trace ID and makes a sampling decision based on the data contained in the trace.
+For instance, inspecting if a trace contains an error.
+
+One particular challenge of grouping trace data is for multi-instance Agent deployments,
+where spans that belong to the same trace can arrive to different Agents.
+To solve that, in the Agent is possible to distribute traces across agent instances by consistently exporting spans belonging to the same trace to the same agent.
```

Comment:
describing this in detail to better highlight the cost of tail based sampling would be good.

Reply:
Added more insight into the overheads of tail sampling